client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/email-provider-stacked-features.tsx

```diff
@@ -1,6 +1,6 @@
 import { Gridicon } from '@automattic/components';
 import { useTranslate } from 'i18n-calypso';
-import { FunctionComponent } from 'react';
+import { FunctionComponent, MouseEventHandler } from 'react';
```

Comment:
I think these should both be type imports.
```suggestion
import type { FunctionComponent, MouseEventHandler } from 'react';
```